lib/pluginmanager/update.rb

```diff
@@ -8,25 +8,28 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   def execute
     local_gems = gemfile.locally_installed_gems
 
-    if update_all? && !local_gems.empty?
-      error_plugin_that_use_path!(local_gems)
-    else
-      plugins_with_path = plugins_arg & local_gems
-      error_plugin_that_use_path!(plugins_with_path) if plugins_with_path.size > 0
+    if local_gems.size > 0
```

Comment:
active_support, jajajaj! well, rails is another world. at the end is everything mostly the same thing, but done in the ruby side or in the core. 

I asked because for me, size is not that self explanatory, but at the end is personal taste, good to go as it is now :+1: